modules/cppitertools/2.1/presubmit.yml

```
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+    - '@cppitertoolscppitertools'
```

The name of the target to be tested should be `@cppitertools//:cppitertools`
```suggestion
    - '@cppitertoolscppitertools'
```